src/map/skill.cpp

```diff
@@ -16591,6 +16718,9 @@ int skill_check_condition_char_sub (struct block_list *bl, va_list ap)
 					sd->status.party_id && tsd->status.party_id && sd->status.party_id == tsd->status.party_id)
 					p_sd[(*c)++] = tsd->bl.id;
 				return 1;
+			case SOA_TALISMAN_OF_FIVE_ELEMENTS:
+				if (tsd->weapontype1>=W_DAGGER && tsd->weapontype1<=W_2HSTAFF)
+				return 0;
```

Comment:
```suggestion
				if (tsd->weapontype1>=W_DAGGER && tsd->weapontype1<=W_2HSTAFF)
					return 0;
```
This looks like it missing "break;".
Also it should be checked if it would be better to use tsd->weapontype1 != W_FIST instead.